GoneSubscriber/GoneDocumentSubscriber.php

```diff
@@ -97,7 +97,13 @@ public function createRedirects(RemoveEvent $event): void
 
         foreach ($this->getUrls($document) as $url) {
             try {
-                $this->redirectRouteManager->saveByData(['source' => $url, 'statusCode' => 410, 'enabled' => true, 'target' => '']);
+                $this->redirectRouteManager->saveByData([
+                    'source' => $url,
+                    'sourceHost' => null,
```

Comment:
Here we need to set the webspace domain for `pages` aslong as there is no `{host}` replacer set. Else if I have a page `/test` in 2 webspaces and remove both it should redirect to the correct domain.

Reply:
How could I get the correct domain? I think there could be multiple configured domains for a single webspace, right?

Reply:
But not sure if we need to be strict here. I think the worst thing that can happen is that we return a `410` in a webspace that never had a page on that URL, right?

Reply:
2 possibilities:

A. we create a redirect for every configured domain for the current environment
B. create a redirect only for the main / first domain found in the webspace config for the current environment

For me both is okay.

Reply:
we already generate all `urls` here https://github.com/sulu/SuluRedirectBundle/blob/d5b52d4fbb2da0fba34b4d67bb0e481949ed8c85/GoneSubscriber/GoneDocumentSubscriber.php#L158 but parsing away the domain

Reply:
> But not sure if we need to be strict here. I think the worst thing that can happen is that we return a 410 in a webspace that never had a page on that URL, right?

🤔 thats correct maybe we stay with this solution currently then.

Reply:
Thanks! But how can I find out if there was a `{host}` replacer in the URL? 🙈

Reply:
Think thats not longer possible as we don't have a method for it 🤔